mindjunkies/lecture/tests/test_forms.py

```
@@ -1,233 +1,100 @@
-from django.test import TestCase
 from django.core.files.uploadedfile import SimpleUploadedFile
-from django import forms
-from django.utils.text import slugify
-from django.core.exceptions import ValidationError
+from model_bakery import baker
 
-from mindjunkies.courses.models import Course, Module, CourseCategory
-from mindjunkies.lecture.models import Lecture, LecturePDF, LectureVideo
 from mindjunkies.lecture.forms import LectureForm, LecturePDFForm, LectureVideoForm, ModuleForm
+from mindjunkies.courses.models import Module
+from mindjunkies.lecture.models import Lecture, LecturePDF, LectureVideo
```

Ruff (0.8.2)

`mindjunkies.lecture.models.LecturePDF` imported but unused

Remove unused import

(F401)